src/pages/device.tsx

```diff
@@ -1,19 +1,221 @@
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .row {
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+  }
+`
+
+const LinkWrapper = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  flex-wrap: wrap;
+
+  padding-bottom: 64px;
+`
+
+const SecondaryLink = styled(Link)`
+  text-decoration: none;
+  background: ${colors.gray.light};
+  border: 3px solid #eee;
+  width: 150px;
+  padding-top: 50px;
+  padding-bottom: 40px;
+  margin-bottom: 12px;
+  border-radius: 8px;
+  display: flex;
+  justify-content: center;
+
+  transition: all 0.2s ease;
+
+  &:hover {
+    transform: translateY(-2px);
+    border: 3px solid #ddd;
+    transition: all 0.2s ease;
+  }
+
+  &:focus {
+    transform: translateY(-2px);
+    border: 3px solid #ddd;
+    transition: all 0.2s ease;
+  }
+
+  .innerWrapper {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+`
 
 const Device: React.FC = () => {
   const intl = useIntl()
+  const [OS, setOS] = useState("")
+  const [loading, toggleLoading] = useState(true)
+
+  useEffect(() => {
+    const userOS = window.navigator.platform.toLocaleLowerCase()
+
+    // Not tested for ios, android, windows
+    if (userOS.includes("mac")) {
+      setOS("mac")
+      toggleLoading(false)
+    } else if (userOS.includes("iphone") || userOS.includes("ipad")) {
+      setOS("ios")
+      toggleLoading(false)
+    } else if (userOS.includes("android")) {
+      setOS("android")
+      toggleLoading(false)
+    } else {
+      setOS("windows")
+      toggleLoading(false)
+    }
+  })
+
+  const renderIcon = () => {
+    let icon
+    if (OS === "android") {
+      icon = <AndroidIcon />
+    } else if (OS === "mac") {
+      icon = <MacIcon />
+    } else if (OS === "ios") {
+      icon = <IosIcon />
+    } else {
+      icon = <WindowsIcon />
+    }
+    return icon
+  }
+
+  const renderDeviceOS = () => {
+    let deviceOS
+
+    if (OS === "android") {
+      deviceOS = "Android"
+    } else if (OS === "mac") {
+      deviceOS = "Mac"
+    } else if (OS === "ios") {
+      deviceOS = "iPhone / iPad"
+    } else {
+      deviceOS = "Windows"
+    }
+    return deviceOS
+  }
+
   return (
     <IndexLayout>
       <Page step1Complete>
         <Container>
-          <Header1>{intl.formatMessage({ id: "title" })}</Header1>
-          <Link to="/service/">{intl.formatMessage({ id: "goto-page" })} 2</Link>
+          {loading ? (
+            <Display>Loading...</Display>
+          ) : (
+            <>
+              <div style={{ marginBottom: 40 }}>
+                <Display>{intl.formatMessage({ id: "devicepageTitle" })}</Display>
+              </div>
+              <div style={{ marginBottom: 16 }}>
+                <Header2>{intl.formatMessage({ id: "currentDeviceTitle" })}</Header2>
+              </div>
+              <PrimaryLink to={`/${OS}/`}>
```

Comment:
I think this Link has to come from the intl package, otherwise it will probably not work on other languages